### PR TITLE
Fixed flake8 errors and updated CHANGELOG.md

### PR DESCRIPTION
**Pull request by AlejandroRG31**
Fixed all flake8 errors except those from importers and exporters to avoid conflicts when merging

Updated CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## master
 
+## [0.4] - 2020-12-22
+- [added] Multiple lexicons can be save and stored (source and destination languages are unique together)
+- [changed] Now there can not be a duplicated word in a lexicon. (unique together lexicon and term)
+- [changed] Avoid repeated entries. (unique together word, translation and variation) 
+- [changed] Importation require to be passed a lexicon_code of an already existing lexicon.
```
Comment (dunetna) [inline CHANGELOG.md]: Typo: Importation **requires** to be passed a lexicon_code of an already existing lexicon.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- [changed] API: View WordViewSet requires a parameter l (lexicon_code) to perform search and near methods.
+- [added] Admin panel and an url_admin_panel in Word model to access it directly.
```
Comment (dunetna) [inline CHANGELOG.md]: Typo? Admin panel **and** an url_admin_panel in Word model to access it directly.

File: runtests.py [hunk 3]
```diff
@@ -142,11 +139,11 @@ def no_available_apps(self):
         'fields.W342',  # ForeignKey(unique=True) -> OneToOneField
     ]
 
-     # Load all the ALWAYS_INSTALLED_APPS.
+    # Load all the ALWAYS_INSTALLED_APPS.
     django.setup()
 
     # Load all the test model apps.
-    test_modules = get_test_modules()
+    # test_modules = get_test_modules()
```
Comment (dunetna) [inline runtests.py]: Could you remove this unused code?

File: CHANGELOG.md (hunk 1 above)
Comment (AlejandroRG31) [reply CHANGELOG.md]: Fixed, thank you for finding it

File: CHANGELOG.md (hunk 2 above)
Comment (AlejandroRG31) [reply CHANGELOG.md]: Rewritten to be more accurate

File: runtests.py (hunk 3 above)
Comment (AlejandroRG31) [reply runtests.py]: Removed, thank you

**Review by slamora — APPROVED** [review]

**Pull request merged**